src/content/config.ts

```diff
@@ -26,9 +26,19 @@ const events = defineCollection({
                     end: z.date().optional(),
                 })
                 .optional(),
-            locationName: z.string().optional(),
-            locationURL: z.string().url().or(z.string().startsWith('#')).or(z.array(z.string().url())).optional(),
-            locationLatLng: z.array(z.number(), z.number()).optional(),
+            locations: z.array(
+                z.object({
+                    locationName: z.string().optional(),
+                    locationURL: z.string().url().or(z.string().startsWith('#')).or(z.array(z.string().url())).optional(),
+                    locationLatLng: z.array(z.number(), z.number()).optional(),
+                    // Fields more relevant for online events
+                    slides: z.array(z.string()).optional(),
```

Comment:
```suggestion
                    slides: z.array(z.string().url()).optional(),
```
also for the rest, so we have automatic URL validation